support-frontend/assets/helpers/subscriptionsForms/submit.ts

```diff
@@ -289,6 +292,21 @@ function onPaymentAuthorised(
 				priceForQuantumMetric,
 				billingPeriod,
 			);
+
+			// track conversion with GTM
+			successfulSubscriptionConversion();
+			if (inThreeTierTestVariant) {
+				/**
+				 * track S+ with GTM, digital split price unavailable from config
+				 * so discounted digitalPlusPrint price applied
+				 **/
+				successfulContributionConversion(
+					digitalPlusPrintPriceDiscounted,
```

Comment:
`digitalPlusPrintPriceDiscounted` will be the combined price of GW and S+? I'm concerned we leave ourselves open to over reporting the Conversion value with this tag as is, as `successfulContributionConversion` should only report the S+ price.

Currently we don't send a price with the GW price in `successfulSubscriptionConversion`, _however_ the stakeholder has requested we do this. The amount we send `successfulSubscriptionConversion` would therefore need to be the GW price, and the amount sent in `successfulContributionConversion` would need to be the S+ price.

Lets catch-up to discuss options.